update.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 #
-# Copyright 2019 JanusGraph Authors
+# Copyright 2019 JanusGraph-Docker Authors
```

Comment:
(nitpick) Is their reason to change the copyright?

Reply:
My thinking on this was that this is a different repo and hence _technically_ a separate subproject, with its own list of AUTHORS and CONTRIBUTORS and hence copyright holders, but perhaps this is too pedantic, so I'm happy to revert this change. It's also orthogonal to the other big change of moving copyright headers to the templates, so we can revisit this (or not) at a different time.

Reply:
Pushed a branch update to revert the copyright line to its original format.